### ISSUE TITLE
Fire modal stretched too long

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#5133]
I clicked the fire icon and this is what the modal looks like:

<img width="860" alt="Screen Shot 2022-01-22 at 9 44 18 AM" src="https://user-images.githubusercontent.com/5498878/150597722-1d758795-dc47-4130-a478-0d134a7527bd.png">

It's the same if I use the arrows to look at other posts. It's the same on Firefox and Chrome on my Macbook.

**Comment by Memo-Es** [comment]
@SudharakaP it s hould be fixed in height and centered

**Comment by SudharakaP** [comment]
> @SudharakaP it s hould be fixed in height and centered

@Memo-Es : The problem here is that if there's an update with a video, we wanted to show the whole video right (rather than blurring the video in the middle). So then another update could be small and therefore will have too much space on top and bottom. Something like this? 

![vertical-align-wierd](https://user-images.githubusercontent.com/12435965/150747952-632c731d-c266-4703-8b78-dfb4cf6bdeea.gif)

**Comment by Betree** [comment]
I'm not convinced by the dynamic height approach implemented in https://github.com/opencollective/opencollective-frontend/pull/7365. Combined with the sliding animation, everything is just moving too much and it's hard to follow (that's just my personal impression).

On the suggestion from https://github.com/opencollective/opencollective/issues/5133#issuecomment-1019840653, these empty spaces look strange. I don't think we want to center the content vertically (I think @Memo-Es was talking about centering the modal, not the content).

![image](https://user-images.githubusercontent.com/1556356/150952765-7e7e2720-fb4a-44f5-9023-6bd01c16170a.png)

Maybe we should stick with the fixed height approach as suggested above. We can eventually display more text for updates (aka make the modal taller) if we want to make sure the size stays the same whether there's a video or not.

**Comment by alanna** [comment]
The problem is if there's a video but it doesn't fit, it was not displaying the video being there at all, just a blank space. Can the video be shown to the extent it fits, like the top half it or whatever, so users know it's there?

**Comment by Betree** [comment]
> The problem is if there's a video but it doesn't fit, it was not displaying the video being there at all, just a blank space. Can the video be shown to the extent it fits, like the top half it or whatever, so users know it's there?

Yes, my proposal above is about keeping that fix for videos while also preserving the fixed height approach. It's a nice experience when users don't have to leave the modal to see the video (when it's at the top).

**Comment by SudharakaP** [comment]
> > The problem is if there's a video but it doesn't fit, it was not displaying the video being there at all, just a blank space. Can the video be shown to the extent it fits, like the top half it or whatever, so users know it's there?
> 
> Yes, my proposal above is about keeping that fix for videos while also preserving the fixed height approach. It's a nice experience when users don't have to leave the modal to see the video (when it's at the top).

@Betree : I've looked at this again and the root cause for this is that sometimes (especially when there's an image in the update) the content takes some time to load and then the `contentRef?.current?.clientHeight` is initially 0 making the `isCollapsed` set to `false`. We need to add a loading placeholder so that we only show the content when `clientHeight` takes a valid value. I've adjusted the PR to account for this. This should fix this issue and also preserve the fixed height. 😉

**Comment by Memo-Es** [comment]
@SudharakaP We shouldn't have these types of issues, the design was intended to have both media content and a read more option if it also includes a text description (In which case it would open a scroll area).
The video thumbnail that I see in the first comments looks too large. Let'make sure to have a fixed height for the modal. All types of updates should fit.

<img width="550" alt="Screen Shot on 2022-01-26 at 10-13-34" src="https://user-images.githubusercontent.com/49539954/151135379-e3ff4366-c18b-4c12-97ac-f263872932d8.png">

**Comment by alanna** [comment]
FYI this is blocking me posting a :fire: for invite org to submit expense, because it won't look right.
